src/Sarif/Autogenerated/Region.cs

```diff
@@ -39,24 +39,32 @@ public virtual SarifNodeKind SarifNodeKind
         /// The line number of the first character in the region.
         /// </summary>
         [DataMember(Name = "startLine", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue(0)]
```

Comment:
>[DefaultValue(0)] [](http://example.com/codeflow?start=8&length=17)

this file is generated from the RTM6 json schema, please see my comment on that file in this PR.

Reply:
Right, so we're making the same mistake though! '0' is a default value that effectively means it isn't there, so we should have a nullable type here, again.

This is more painful from an API use standpoint, maybe too painful! But everything would be consistent. 

I do see from the spec that the region class in particular has complex/unusual default value semantics...